docs/UserGuide.md

```diff
@@ -100,6 +100,14 @@ Format: `add n\NAME p\PHONE_NUMBER e\EMAIL a\ADDRESS fi\FINANCIALINFO s\SOCIALME
 A tag can be added with or without a value. For example, `t\client` and `t\insurance:No` are both valid tags.
 </box>
 
+<box type="warning" seamless>
+**Tag constraints:**<br>
+* Only alphanumeric characters (alphabets and numbers) are supported in tags<br>
+* Multiple of the same tags are not supported for a single contact<br>
+* Negative numbers are not supported in tag names<br>
```

Comment:
Is it correct to say tag names? Should it be "Negative numbers are not supported in tag values"? Or am I mistaken?

Reply:
Im blind, thanks for catching that